### DISCUSSION TITLE
ideas

### DISCUSSION THREAD

**Discussion by dragonfgamer6665** [danielkrupinski/Osiris#2799] [Ideas]
add custom skybox add night mod add full bright add legit aa or improve the aa  please

**Comment by godasuna** [comment]
he already has  tho


Am Do., 15. Apr. 2021 um 04:04 Uhr schrieb dragonfgamer6665 <
***@***.***>:

**Reply by dragonfgamer6665** [reply]
not night mod and full bright and legit aa

**Reply by dragonfgamer6665** [reply]
I can't see custom skybox but people there is

**Comment by godasuna** [comment]
He already has tho

Am Do., 15. Apr. 2021 um 04:04 Uhr schrieb dragonfgamer6665 <
***@***.***>:

**Comment by godasuna** [comment]
Nightmode is when you select world color as gray. There is a skybox changer
in the visuals tab and Legit aa is when you dont edit the aa settings.
You'r player bodel will be moving sideways every so slightly.

**Reply by dragonfgamer6665** [reply]
when I mean custom skybox is you can put your own skybox in the cheat

**Reply by KurastikO** [reply]
You can by altering the source code
